build.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
What is the reasoning behind this change?

Reply:
sh is more widely available, more specifically bash is not available in an alpine docker image